analytics.go

```diff
@@ -2,8 +2,11 @@ package pagerduty
 
 import (
 	"context"
+	"fmt"
 )
 
+const baseURL = "/analytics/metrics/incidents"
```

Comment:
Since `constants` are local to the package and not just this file, would you be able to make this constant name a little more precise? Maybe `analyticsBaseURL`?